README.md

```diff
@@ -45,9 +45,10 @@ class Foo
 end
 ```
 
-If `#as_json` is called without the `:presenter_action` options, or if the
-presenter class doesn't exist, this delegates to the original model class
-definition of `#as_json`
+If `#as_json` is called without the `:presenter_action` options, or if
+the presenter class doesn't exist, this delegates to
```

Comment:
Also, if I'm reading this correctly, if you don't include the `:presenter_action` option, this will continue to be handled by the model's `#as_json` method, because in the `AsJsonPresentable` module that gets mixed into your model class, we still have this implementation of `#as_json`:
```ruby
  def as_json(options=nil)
    if options && options[:presenter_action] && json_presenter_class
      json_presenter_class.new(self).as_json(options)
    else
      super(options)
    end
  end
```
Which I think has to remain in place, because otherwise you get in an endless loop.

So it looks to me like all this change does is makes it so if you pass in a `:presenter_action`, but it's invalid, it will call `#as_default_json`.

So here's what I think: I like the idea of having a base implementation of `#as_default_json` that just delegates to the resource, but could be overridden in certain presenters. It lets us use the same interface for various models with a solid base implementation. But I don't think it serves any purpose to try to fall back to that `#as_default_json` if you pass in an invalid `:presenter_action`